src/chrome-service.ts

```diff
@@ -143,8 +156,8 @@ export class ChromeService {
       const page = await browser.newPage();
       const cleanup = () => this.config.keepAlive ?
         _.attempt(() => {
-          page.close();
-          this.chromeSwarm.push(Promise.resolve(browser));
+          this.reuseChromeInstance(browser);
+          debug(`Added to chrome swarm: ${this.chromeSwarm.length} online`);
```

Comment:
Thanks for adding the debug statements, really helps (especially when your awoken in the middle of the night because of an issue)